### PR TITLE
Different changes to work on pacemaker stability

### PR DESCRIPTION
**Pull request by insatomcat**
pacemaker: modify service dependencies
Change pacemaker dependencies to add rbdmap, ceph and libvirt, so that the guests can be cleanly migrated or shutdown before pacemaker shuts down.

pacemaker: allow to write in systemd runtime files
Pacemaker needs to be able to write to systemd runtime dependencies folder so that the VirtualDomain resource agent can set the required dependencies.

systemd-journald: add log persistence
This allow the journald logs to be persisted on disk so that it becomes possible to investigate a previous shutdown process journal.

pacemaker: modify shutdown timeout to 2min
in case of anything preventing pacemaker to cleanly shutdown (guest not migrating properly for instance), the default timeout before systemd forces the shutdown is 30min. It's way too long, and waiting 2 min for pacemaker is enough.

Signed-off-by: insatomcat <florent.carli@rte-france.com>

### CODE REVIEW

File: recipes-cgl/pacemaker/files/pacemaker.service [hunk 1]
```diff
@@ -33,11 +33,14 @@ Requires=corosync.service
 After=create-var-run-resource-agents.service create-pacemaker-log-dir.service
 Wants=create-var-run-resource-agents.service create-pacemaker-log-dir.service
 
+#necessary for the live migration to work when a host is shutdown or rebooted
```
Comment (ebail) [inline recipes-cgl/pacemaker/files/pacemaker.service:36]: shutdowned

**Review by ebail — CHANGES_REQUESTED** [review]
Hi Thanks for your work.

I would rework the commit message. It might sound picky but I believe it is important for the history of the project.

As described in https://github.com/seapath/.github/blob/main/CONTRIBUTING.md#styleguides.

I would commit:

pacemaker: modify service dependencies 

Change pacemaker dependencies to to X, Y and Z.


Thanks

File: recipes-cgl/pacemaker/files/pacemaker.service (hunk 1 above)
Comment (insatomcat) [reply recipes-cgl/pacemaker/files/pacemaker.service]: changed

**Review by dupremathieu — APPROVED** [review]

**Review by ebail — APPROVED** [review]
LGTM, Thanks

**Pull request merged**